weasyl/login.py

```diff
@@ -283,9 +283,12 @@ def create(form):
         #   let the already registered user know this via email (perhaps they forgot their username/password)
         query_username_login = d.engine.scalar("SELECT login_name FROM login WHERE email = %(email)s", email=email)
         query_username_logincreate = d.engine.scalar("SELECT login_name FROM logincreate WHERE email = %(email)s", email=email)
-        emailer.send(email, "Weasyl Account Creation - Account Already Exists", d.render(
-            "email/email_in_use_account_creation.html", [query_username_login or query_username_logincreate]))
-
+        if query_username_login:
+            emailer.send(email, "Weasyl Account Creation - Account Already Exists", d.render(
+                "email/email_in_use_account_creation.html", [query_username_login]))
+        else:
+            emailer.send(email, "Weasyl Account Creation - Account Already Exists", d.render(
+                "email/email_in_use_account_creation_in_progress.html", [token, query_username_logincreate]))
```

Comment:
Thanks for the PR – this is a very important feature!

It looks like the token being sent here will correspond to the invalid `logincreate` record created on line 271. Do we need to look up the real one, or did you test it and find out that the one that was intended to be invalid accidentally works in this situation? :D